packages/explorer/package.json

```
@@ -5,7 +5,10 @@
   "description": "TypeScript types for Enonic XP",
   "typings": "index.d.ts",
   "files": [
-    "index.d.ts"
+    "index.d.ts",
+    "client.d.ts,
```

This appears to be missing a `"`.

```suggestion
    "client.d.ts",
```